guillotina_amqp/utils.py

```diff
@@ -101,3 +119,42 @@
     return await add_task(
         _run_object_task, get_dotted_name(callable), get_content_path(ob), *args,
         _request=_request, _retries=_retries, **kwargs)
+
+
+class TimeoutLock(object):
+    """Implements a Lock that can be acquired for """
+    def __init__(self, worker_id):
+        self._lock = threading.Lock()
+        self.worker_id = worker_id
+        self._thread = None
```

Comment:
What is `self._thread` for here?

Reply:
oh, nothing. just legacy code from an interemediate implementation. removing :+1: 

thanks!!